node_modules/oae-admin/tenants/css/tenants.css

```diff
@@ -13,32 +13,89 @@
  * permissions and limitations under the License.
  */
 
-#tenants-container .table-striped td.tenants-visit {
-    width: 20px;
+#tenants-search {
+    margin-bottom: 5px;
+    text-align: right;
 }
 
-#tenants-container .table-striped td.tenants-visit i {
+#tenants-search > form {
+    display: inline-block;
+    margin-bottom: 5px;
+    min-width: 300px;
+}
+
+#tenants-table li {
+    background-color: #FFF;
+    border-bottom: solid 1px #DDD;
+}
+
+#tenants-table li > div {
+    line-height: 35px;
```

Comment:
This is causing too much space when the tenant name drops to multiple lines:

<img width="134" alt="screen shot 2015-09-10 at 09 51 57" src="https://cloud.githubusercontent.com/assets/109850/9795724/21017474-57a7-11e5-93bf-167a9c7d183d.png">